examples/prometheus-operator/tidb-cluster.yaml

```diff
@@ -5,22 +5,27 @@ metadata:
 spec:
   version: v3.0.8
   timezone: UTC
-  pvReclaimPolicy: Delete
+  pvReclaimPolicy: Retain
+  enableDynamicConfiguration: true
+  configUpdateStrategy: RollingUpdate
   pd:
     baseImage: pingcap/pd
     replicas: 3
     requests:
       storage: "1Gi"
+    mountClusterClientSecret: true
     config: {}
   tikv:
     baseImage: pingcap/tikv
     replicas: 3
     requests:
       storage: "1Gi"
+    mountClusterClientSecret: true
```

Comment:
`mountClusterClientSecret` is only for PD and TiKV and should only be added when `spec.tlsCluster.enabled: true`.